server/services/permissionsService.test.ts

```diff
@@ -0,0 +1,785 @@
+
+        const permissions = await service.getOverviewPermissions(user, prisoner)
+
+        expect(permissions.incentives).toEqual({
+          view: true,
+        })
+      })
+    })
+
+    describe('category', () => {
+      it('should return edit false if user doesnt have appropriate role', async () => {
+        const permissions = await service.getOverviewPermissions(prisonUserMock, PrisonerMockDataA)
+
+        expect(permissions.category).toEqual({
+          view: false,
+          edit: false,
+        })
+      })
+
+      it.each([
+        Role.CreateRecategorisation,
+        Role.ApproveCategorisation,
+        Role.CreateRecategorisation,
+        Role.CategorisationSecurity,
+      ])('should return edit true if user has %s role', async role => {
+        const user = { ...prisonUserMock, userRoles: [role] }
+
+        const permissions = await service.getOverviewPermissions(user, PrisonerMockDataA)
+
+        expect(permissions.category).toEqual({
+          view: false,
+          edit: true,
+        })
+      })
+    })
+
+    describe('calculateReleaseDates', () => {
+      it('should return edit false if user doesnt have appropriate role', async () => {
+        const permissions = await service.getOverviewPermissions(prisonUserMock, PrisonerMockDataA)
+
+        expect(permissions.category).toEqual({
+          view: false,
+          edit: false,
+        })
+      })
+
+      it('should return edit true if user has ReleaseDatesCalculator role', async () => {
+        const user = { ...prisonUserMock, userRoles: [Role.ReleaseDatesCalculator] }
+
+        const permissions = await service.getOverviewPermissions(user, PrisonerMockDataA)
+
+        expect(permissions.calculateReleaseDates).toEqual({
+          view: false,
+          edit: true,
+        })
+      })
+    })
+
+    describe('caseNotes', () => {
+      it('should return edit true if prisoner is in users case loads', async () => {
+        const permissions = await service.getOverviewPermissions(prisonUserMock, PrisonerMockDataA)
+
+        expect(permissions.caseNotes).toEqual({
+          view: false,
+          edit: true,
+        })
+      })
+
+      it('should return edit false if prisoner is not in users case loads', async () => {
+        const user = {
+          ...prisonUserMock,
+          caseLoads: [{ caseLoadId: 'LEI', currentlyActive: true } as CaseLoad],
+          userRoles: [Role.GlobalSearch],
+        }
+
+        const permissions = await service.getOverviewPermissions(user, PrisonerMockDataA)
+
+        expect(permissions.caseNotes).toEqual({
+          view: false,
+          edit: false,
+        })
+      })
+
+      it('should return edit true if user has GlobalSearch and PomUser roles', async () => {
+        const user = {
+          ...prisonUserMock,
+          userRoles: [Role.GlobalSearch, Role.PomUser],
+          caseLoads: [{ caseLoadId: 'LEI', currentlyActive: true } as CaseLoad],
```

Comment:
Or I wonder if the intent would be clearer if it was `caseLoadId: 'SOMETHING_ELSE'` since at first glance we can't see that it's actually different from the prisoner mock data 🤷